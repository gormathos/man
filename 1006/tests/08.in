C1;C2;C3;C4
65;42;24;49
34;15;65;11
38;77;52;76
90;57;51;77
77;90;57;82
40;79;74;35
66;87;87;89
37;33;57;80
62;31;11;72
18;53;17;17
75;61;62;48
23;54;54;15
63;89;33;15
74;84;33;19
10;49;88;69
12;27;71;22
64;61;18;29
16;13;82;91
19;50;84;28
27;12;22;31
78;16;46;78
48;47;77;26
57;91;61;52
54;29;44;87
34;72;26;67
64;10;57;57
28;25;22;14
51;47;21;12
23;90;14;26
80;21;17;31
75;87;32;84
48;76;17;38
85;46;39;20
59;63;89;36
19;82;30;15
10;84;29;65
17;12;16;29
36;26;44;21
45;22;60;92
79;14;70;74
11;73;39;10
87;50;75;27
79;83;48;38
69;68;22;85
85;89;48;19
53;20;49;79
75;91;10;58
58;86;61;92
49;91;35;64
35;74;53;16
27;25;56;86
79;27;88;70
85;78;60;88
64;10;23;10
39;63;30;44
10;15;60;56
37;67;85;89
74;54;41;42
31;17;26;59
86;67;47;52
56;65;72;55
21;65;19;27
76;20;15;74
48;68;32;65
89;84;36;14
13;89;33;79
50;92;65;35
56;35;77;59
23;10;80;56
36;72;46;26
42;49;89;52
48;14;89;20
28;66;23;75
87;76;71;65
84;54;77;60
14;88;24;92
34;46;78;37
74;66;16;18
82;65;41;27
43;57;11;38
67;30;38;84
89;31;34;47
43;52;36;69
19;62;43;43
17;69;88;47
42;70;32;43
18;25;51;54
85;37;85;45
25;49;12;24
82;56;38;68
41;69;47;77
78;69;71;49
29;10;92;15
74;55;27;44
83;88;43;54
42;69;59;83
80;73;85;82
59;86;53;42
71;28;42;76
18;54;64;24
