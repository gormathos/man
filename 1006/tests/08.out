20384
92
10
